backend/tests/integration/mailing/test_api_jobs.py

```diff
@@ -54,13 +55,15 @@ def test_render_jobs_weekly_campaign_job_record_template_only(create_jobs):
 
 @pytest.mark.vrc()
 def test_subscirbe_to_the_list(inspire_app, vcr_cassette):
-    result = subscribe_to_jobs_weekly_list("luke@cage.com", "Luke", "Cage")
+    subscribe_to_jobs_weekly_list("luke@cage.com", "Luke", "Cage")
     assert vcr_cassette.all_played
 
 
-def test_subscirbe_to_the_list_with_invalid_email(inspire_app):
+@pytest.mark.vrc()
+def test_subscirbe_to_the_list_with_invalid_email(inspire_app, vcr_cassette):
     with pytest.raises(ValueError):
         subscribe_to_jobs_weekly_list("luke", "Luke", "Cage")
+    assert vcr_cassette.all_played
```

Comment:
maybe we should have something like this in the `conftest`? we do for `orcid` tests.